docs/docs-source/docs/modules/get-started/pages/deploy-to-k8s-minikube.adoc

```diff
@@ -0,0 +1,136 @@
+= Deploy the Example to Minikube
+
+
+include::ROOT:partial$include.adoc[]
+
+This guide shows how to deploy the Cloudflow application using the Minikube `registry` addon.
+For more detailed instructions please refer to the official https://minikube.sigs.k8s.io/docs/handbook/pushing/#4-pushing-to-an-in-cluster-using-registry-addon[Pushing images] documentation of Minikube.
+
+Unfortunately Cloudflow does not support changing the `imagePullPolicy` of application images yet (currently fixed to `imagePullPolicy: Always`),
+which is required by other (easier) options for pushing images to Minikube.
+
+== Setup
+
+ * Enable insecure registries in Minikube.
+ * Enable the `registry` Minikube addon.
+ * Configure Docker ot use the insecure registry.
+ * Configure the target environment.
+ * xref:administration:installing-cloudflow.adoc[Install Cloudflow] on Minikube (if you haven't done so already)
+
+We will assume that the Minikube VM has an ip from the `192.168.39.0/24` subnet.
+You can verify this once Minikube is started by using `minikube ip`.
+
+Start Minikube with insecure registries and enable the `registry` addon:
+
+   minikube start --kubernetes-version=latest --insecure-registry "10.0.0.0/24,192.168.39.0/24"
+   minikube addons enable registry
+
+Configure Docker to use the Minikube registry as insecure registry by adding it to
+the `daemon.json` file, whose default location is `/etc/docker/daemon.json` on Linux.
+If the `daemon.json` file does not exist, create it.
```

Comment:
Please add some information on how to do this for Windows / Mac, some details here: https://docs.docker.com/registry/insecure/#deploy-a-plain-http-registry (and probably smart to link to this page)